verification/certifyKey_test.go

```diff
@@ -14,19 +15,43 @@ import (
 	"github.com/zmap/zlint/v3/lint"
 )
 
-// This file is used to test the certify key command by using a simulator
+// This file is used to test the certify key command by using a simulator/emulator
+
+func GetTestTarget_CertifyKey() ([]TestDPEInstance, error) {
+
+	// Added dummy support for emulator
+	support_needed := []string{"AutoInit", "X509"}
+
+	var instances []TestDPEInstance
+
+	if testTargetType == EMULATOR {
```

Comment:
If testTargetType == EMULATOR and the requested support configuration isn't supported by the emualtor, return an error.

Reply:
Addressed